satellite/overlay/service.go

```diff
@@ -47,6 +47,8 @@ type DB interface {
 	// SelectAllStorageNodesUpload returns all nodes that qualify to store data, organized as reputable nodes and new nodes
 	SelectAllStorageNodesUpload(ctx context.Context, selectionCfg NodeSelectionConfig) (reputable, new []*SelectedNode, err error)
 
+	// GetAllNodeIDs returns all node IDs
+	GetAllNodeIDs(ctx context.Context) ([]storj.NodeID, error)
```

Comment:
i get the urgency but we've really been trying hard to use pagination where possible, and dbx supports pagination.